include/picongpu/algorithms/AssignedTrilinearInterpolation.hpp

```diff
@@ -52,64 +52,69 @@ namespace picongpu
         /** Does a 3D trilinear field-to-point interpolation for
          * arbitrary assignment function and arbitrary field_value types.
          *
-         * @tparam T_AssignmentFunction function for assignment
          * @tparam T_begin lower margin for interpolation
          * @tparam T_end upper margin for interpolation
+         * @tparam T_AssignmentFunctionArray type of the array with shape functors
          *
          * @param cursor cursor pointing to the field
-         * @param pos position of the interpolation point
+         * @param shapeFunctorArray Array with d shape functors, where d is the number of simulation.
```

Comment:
Is `d` dimensionality? Then could `simDim` be used?

Reply:
should be the dimensionality of the field